samples/nrf9160/modem_shell/Kconfig

```
@@ -91,6 +91,16 @@ config MOSH_CLOUD
 	help
 	  MQTT connection to nRFCloud.
 
+if NRF_CLOUD_PGPS && !NRF_CLOUD_PGPS_PARTITION
+
+config MOSH_PGPS_MCUBOOT_PARTITION
+	bool "Use mcuboot secondary partition for P-GPS data"
```

This isn't supposed to be configured by the user, so maybe we could remove the prompt?
```suggestion
	bool
```